### PR TITLE
chore: add Github actions status badge

### PR DESCRIPTION
**Pull request by damienwebdev**
Depends on #58

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-
-[![Build Status](https://travis-ci.org/Marak/faker.js.svg?branch=master)](https://travis-ci.org/Marak/faker.js) [![Coverage Status](https://coveralls.io/repos/github/Marak/faker.js/badge.svg?branch=master)](https://coveralls.io/github/Marak/faker.js?branch=master)
-
-[![npm version](https://badge.fury.io/js/faker.svg)](http://badge.fury.io/js/faker)
-
-[![OpenCollective](https://opencollective.com/fakerjs/backers/badge.svg)](#backers)
-[![OpenCollective](https://opencollective.com/fakerjs/sponsors/badge.svg)](#sponsors)
-[![Gitter chat](https://badges.gitter.im/marak/faker.js.png)](https://gitter.im/Faker-js/community)
+[![Chat on Discord](https://img.shields.io/discord/929487054990110771)](https://discord.com/invite/4qDjAmDj4P)
+[![Continuous Integration](https://github.com/faker-js/faker/actions/workflows/ci.yml/badge.svg)](https://github.com/faker-js/faker/actions/workflows/ci.yml)
```
Comment (Shinigami92) [inline Readme.md:4]: Maybe just

```suggestion
[![CI](https://github.com/faker-js/faker/actions/workflows/ci.yml/badge.svg)](https://github.com/faker-js/faker/actions/workflows/ci.yml)
```

Comment (Shinigami92) [reply Readme.md:4]: I mean, the text is really long

Comment (damienwebdev) [reply Readme.md:4]: Interestingly, that text isn't controllable, it's in the SVG

Comment (Shinigami92) [reply Readme.md:4]: So it's the name in the ci.yml ...
Can we change that?

Comment (damienwebdev) [reply Readme.md:4]: I don't see a strong need too unless we plan on having MANY badges.

**Review by Shinigami92 — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Review by griest024 — APPROVED** [review]

**Pull request merged**